opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1340,12 +1340,11 @@ namespace Opm {
                 // Volume factors, densities and viscosities need to be recalculated with the updated rs and rv values.
                 if (ebosSimulator_.episodeIndex() < 0 && vapour_active && liquid_active ) {
 
-                    typedef Opm::CompositionalFluidState<Scalar, FluidSystem> ScalarFluidState;
-                    const ScalarFluidState& fs_updated = ebosSimulator().problem().initialFluidState(cellIdx);
+                    const auto& fs_updated = ebosSimulator().problem().initialFluidState(cellIdx);
 
                     // use initial rs and rv values
-                    Rv[cellIdx] = Opm::BlackOil::getRv_<FluidSystem, Scalar, ScalarFluidState>(fs_updated,  intQuants.pvtRegionIndex());
```

Comment:
it is probably better to leave this like it is: with `Opm::BlackOil::getRv_` it works regardless of whether `fs_updated` features a `Rv()` method or not.

Reply:
(it just calls `Rv()` if it exists, or calculates it using generic quantities otherwise. There should thus not be any performance change.)

Reply:
I hope this part of the code will die very soon when the output is moved to ewoms. The PR is ready, just waiting for this and the PR in ewoms to be merged so I can rebase it on top of it.

Reply:
I agree and I intend to merge as soon as Jenkins is happy. without this change, this PR does not need to wait for the eWoms one anymore, though...